src/templates/static/mt5_signals.jsx

```diff
@@ -0,0 +1,240 @@
+                                                <li>{it.L('From your MT5 trading terminal, click on the <strong>Signals</strong> tab.')}</li>
+                                            </ol>
+                                        </div>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/mt5_statistics.png')} />
+                                            <ol start='2'>
+                                                <li>{it.L('Click on the <strong>My Statistics</strong> tab and select your signal provider.')}</li>
+                                            </ol>
+                                        </div>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/mt5_subscribe2.png')} />
+                                            <ol start='3'>
+                                                <li>{it.L('Click the <strong>Renew</strong> button to renew the service or the <strong>Unsubscribe</strong> button to cancel the subscription.')}</li>
+                                            </ol>
+                                        </div>
+                                    </div>
+                                    <div>
+                                        <p>{it.L('<strong>Note:</strong>')}</p>
+                                        <ol>
+                                            <li>{it.L('You must have an MQL5 community account to subscribe to trading signals. If you don’t have an account yet, please go to [_1] to register.', '<a href="https://www.mql5.com/en/auth_register">MQL5.com</a>')}</li>
+                                            <li>{it.L('You’ll need to fund your MQL5 account to subscribe to paid signals. [_1]', '<a href="https://www.mql5.com/en/articles/302#deposit">Learn how to fund your account</a>')}</li>
+                                            <li>{it.L('You can only subscribe to one signal provider with one Binary.com account at any given time. You can use your signal on up to 3 computers.')}</li>
+                                            <li> {it.L('You won’t be able to trade manually with the same Binary.com MT5 account once you’ve subscribed to a signal.')}
+                                            </li>
+                                        </ol>
+                                        <p>{it.L('Have more questions? Check out the [_1] section on the MQL5 website.', '<a href="https://www.mql5.com/en/forum/10773#q17">FAQ </a>')} </p>
+                                        <div className='center-text gr-padding-20'>
+                                            <a className='button' href='https://app.deriv.com/mt5'><span>{it.L('Go to the MT5 Terminal')}</span></a>
```

Comment:
is this the expected link to redirect clients to?

Reply:
as for now maybe we let it this way , and gonna reconfirm it with the PM and QA

Reply:
have they confirmed yet?

Reply:
Islam told me on slack should be https://www.binary.com/en/user/metatrader.html